script/gitlabci/job_base.yml

```diff
@@ -4,6 +4,8 @@
     ALPAKA_CI_BOOST_BRANCH: "boost-${ALPAKA_BOOST_VERSION}"
     BOOST_LIBRARYDIR: "/opt/boost/${ALPAKA_BOOST_VERSION}/lib"
     ALPAKA_CI_OS_NAME: "Linux"
+    ALPAKA_CI_BUILD_JOBS: "$CI_MAX_PARALLELISM"
+    OMP_NUM_THREADS: "$CI_MAX_PARALLELISM"
```

Comment:
Where are those variables coming from?

Reply:
Thats the right question. I copied it from PIConGPU and there I have the comment that this variable is set in the CI Web interface. SO I used the wrong variable name `CI_CPUS` should be used because this is a variable set by the HZDR CI instances.

Reply:
Updated an I added a short string above the usage where this variable is coming from.